examples/automm/Kaggle-Pawpularity/kaggle_Pawpularity.py

```diff
@@ -0,0 +1,103 @@
+from autogluon.text.automm import AutoMMPredictor
+from sklearn.model_selection import StratifiedKFold
+from sklearn.metrics import mean_squared_error
+import pandas as pd
+import numpy as np
+import torch
+import os
+
+
+def load_data(data_path: str):
+    """
+    Load training and testing datasets and split folds.
+    Parameters
+    ----------
+    data_path
+        The path of training and testing files.
+
+    Return
+    ----------
+    The pure training dataset and the folds split information and the testing dataset.
+    """
+    # Load training dataset.
+    train_df = pd.read_csv(os.path.join(data_path, "train.csv"))
+    train_df.rename(columns={"Id": "Image Path"}, inplace=True)
+    train_df["Image Path"] = train_df["Image Path"].apply(lambda s: os.path.join(data_path, "train", s + ".jpg"))
+
+    # Creat the split information of folds.
+    num_bins = int(np.ceil(2 * ((len(train_df)) ** (1. / 3))))
+    train_df_bins = pd.cut(train_df["Pawpularity"], bins=num_bins, labels=False)
+    train_df_fold = pd.Series([-1] * len(train_df))
+    strat_kfold = StratifiedKFold(n_splits=N_FOLDS, shuffle=True)
+    for i, (_, train_index) in enumerate(strat_kfold.split(train_df.index, train_df_bins)):
+        train_df_fold[train_index] = i
+    train_df_fold = train_df_fold.astype("int")
+
+    # Load testing dataset.
+    test_df = pd.read_csv(os.path.join(data_path, "test.csv"))
+    test_df.rename(columns={"Id": "Image Path"}, inplace=True)
+    test_df["Image Path"] = test_df["Image Path"].apply(lambda s: os.path.join(data_path, "test", s + ".jpg"))
+
+    return train_df, train_df_fold, test_df
+
+
+if __name__ == "__main__":
+    data_path = "./data/"  # The path of the training and testing data.
+    save_path = "./pawpularity-result/result1/result"  # The path of saving the model.
```

Comment:
consider to change the name of the save_path based on the name of the backbone + whether to use fusion or not.